ghcide/src/Development/IDE/Main.hs

```diff
@@ -18,8 +18,8 @@ import           Control.Concurrent.STM.Stats             (dumpSTMStats)
 import           Control.Exception.Safe                   (SomeException,
                                                            catchAny,
                                                            displayException)
-import           Control.Monad.Extra                      (concatMapM, unless,
-                                                           when)
+import           Control.Monad.Extra                      (concatMapM, join,
```

Comment:
I suspect some unused imports here :smile: 
I'd do `cabal clean` and rebuilt to see them all ..